content/project-organization/quality-council/processes/issue-management.md

```diff
@@ -344,45 +362,36 @@ You can personalize the message according to your needs, for example:
 For active contributors, it’s not necessary to add this block:
 
 ```
-If this one is important to you [...], the better PrestaShop becomes for everyone.
+If this one is important to you [...] , the better PrestaShop becomes for everyone.
 ```
 
 Instead, you can add this:
 
 ```
-We're waiting for your PR 🚀
+We’re waiting for your PR 🚀
 ```
 
-###### Regression
-
-If it’s a regression, add this sentence:
-
-```
-I didn't manage to reproduce the issue with PS x.x.x, I only reproduce it with PS x.x.x. So it's a regression!
-```
-
-#### Issue not completed
+### Issue not completed
 
 ```
 Hello [name],
 
-Thank you for submitting this issue. Unfortunately, there is not enough information provided to reproduce it and work on it. **You must follow the template and provide clear and concise steps to reproduce the bug**.
-
-Read more about how we expect the issues to be handled [here](https://build.prestashop-project.org/news/how-bug-reports-are-handled/). Feel free to create a new issue if you can provide further information, we will be glad to investigate it.
+Thank you for submitting this issue. Unfortunately, there is not enough information provided to reproduce it and work on it. **You must follow the template and provide clear and concise steps to reproduce the bug**. 
+Read more about how we expect the issues to be handled [here](https://build.prestashop-project.org/news/2020/how-bug-reports-are-handled/). Feel free to create a new issue if you can provide further information, we will be glad to investigate it. 
 
 Thank you
 ```
 
-#### Issue is a functional feature request
+### Issue is a functional feature request
```

Comment:
That part is deprecated, right? somebody from the project should be informed about it, so that we could transfer it to the Discussions.

Reply:
I leave it to @Robin-Fischer-PS to decide as lead of the Quality Council.

Reply:
This part is indeed Deprecated. The process right now is to convert issue for feature request into a GH Discussion, but we do not have (yet) a template message for that.